modules/configuring-firewall.adoc

```diff
@@ -38,6 +38,10 @@ There are no special configuration considerations for services running on only c
 |443, 80
 |Provides {op-system-first} images
 
+|`rhc4tp-prod-z8cxf-image-registry-us-east-1-evenkyleffocxqvofrk.s3.dualstack.us-east-1.amazonaws.com`
+|443
+|Provides access to container images hosted on `registry.connect.redhat.com`
+
```

Comment:
revert this section as it already represents in `Allowlist the following URLs for optional third-party content`

Reply:
Addressed